action.yml

```diff
@@ -15,7 +15,11 @@ inputs:
     default: ${{ github.token }}
   skip-commit-verification:
     type: boolean
-    description: 'If true, the action will not expect Dependabot commits to be verified. This should be set as `true` in GHES environments.'
+    description: 'If true, the action will not expect Dependabot commits to be verified. This should be set as `true` in GHES environments'
+    default: false
+  skip-verification:
+    type: boolean
+    description: 'If true, the action will not validate the user or the commit verification status'
```

Comment:
I'd prefer to remove `skip-commit-verification` in favor of `skip-verification`, but given https://github.com/dependabot/fetch-metadata/issues/332#issuecomment-1500509027 there are probably some GHES users who are forced to set `skip-commit-verification`, but still want the user verification in order to avoid any accidental CI runs.

That'd also be a breaking change, so even if we later change our minds, we'd still want this existing `skip-verification` to exist for a few releases as an easy migration path... 

And anyone who sets `skip-verification` no longer needs to set `skip-commit-verification`.